src/main/java/address/ui/Ui.java

```diff
@@ -4,18 +4,23 @@
 import address.browser.BrowserManager;
 import address.controller.MainController;
 import address.model.ModelManager;
+import address.model.UserPrefs;
 import address.util.Config;
 import javafx.scene.control.Alert;
 import javafx.stage.Stage;
+import javafx.util.Pair;
+import org.apache.commons.lang3.tuple.ImmutablePair;
```

Comment:
Yes, I'd rather minimize additional libraries unless we really need them.
BTW, don't you need to save the position as well?